.orchestra/config/components/revng-python-dependencies.yml

```diff
@@ -37,6 +38,7 @@ requirements: |-
 
   # Applications/Script dependencies
   ariadne[asgi-file-uploads]>=0.20.1
+  boto3
```

Comment:
Is this used by the `revng` component or by `.orchestra/support/mass-testing/`?

Reply:
Right now by `mass-testing*`, but I expect `revng` to use it soon either way (e.g. pypeline). It seemed it did not warrant creating another component for it.